src/appleseed-max/plugin.cpp

```diff
@@ -135,6 +135,8 @@ extern "C"
         static const wchar_t PluginImplDLLFilename[] = L"appleseed-max2017-impl.dll";
 #elif MAX_RELEASE == MAX_RELEASE_R20
         static const wchar_t PluginImplDLLFilename[] = L"appleseed-max2018-impl.dll";
+#elif MAX_RELEASE == MAX_RELEASE_R21
```

Comment:
Not related to Embree so I'd say this is in the wrong commit, but we can live with that. However are there not other changes required to support Max 2019? At least we need to provide a new VS solution... All this work should probably be done in another commit or even PR.

Reply:
I can only test Max 2019, so I need to have this line to compile it. We can just comment it out on master for the time being. I have a VS solution of course, copied the one for Max 2018 and modified the link to the SDK to point to 2019 version. I do the development on a local copy of the project folder which gets pushed to GitHub and copy just the source files I had changed to it. This way I wanted to avoid that visual studio project files with project specific settings to my personal builds gets pushed to GitHub. I'm not sure this is the best way to do it. I haven't worked on a GitHub project before.

Reply:
@dictoon max2018 and 2019 have compatible sdk-s. But we still need a separate solution, right? To copy things correctly and everything..

Reply:
Yes we need a separate solution.